src/utils/testing-utils/utils.ts

```diff
@@ -15,17 +21,17 @@ export const clickAndNavigate = async (
   page: Page,
   selector: string,
   waitFor?: string
-) => {
+): Promise<string> => {
   await Promise.all([
     page.waitForNavigation({
-      waitUntil: ["load", "domcontentloaded", "networkidle0"],
+      waitUntil: ["load", "domcontentloaded"],
```

Comment:
It's a questionable change. The "networkidle0" is important in situations when the page fetches elements that are expected to be loaded completely by the test.
The downside of it is it's pron to timeouts.
Unfortunately the Puppeteer doesn't have auto-wait feature that Playwright has.
The "networkidle0" was added in our code to prevent the tests from failing while indirect navigation happens. 
Please note, it's used for indirect navigation only in the dashboard tests.

Reply:
e2e testing doesn't rely on http requests. We don't test layers, don't need to wait when data is loaded. The only case is comparison button. It is enabled after layers are loaded. This test is skipped for long time.
Probably, this is a good approach to fix this test